restcomm/restcomm.http/src/main/java/org/mobicents/servlet/restcomm/http/AbstractEndpoint.java

```diff
@@ -99,6 +101,46 @@ protected URI getUrl(final String name, final MultivaluedMap<String, String> dat
         return uri;
     }
 
+    protected Integer getInteger(final String name, final MultivaluedMap<String, String> data) {
+        Integer integer = null;
+        if (data.containsKey(name)) {
+            integer = new Integer(data.getFirst(name));
```

Comment:
It's definitely a better choice @jaimecasero (again, should be followed for the rest of the methods that stick with this criteria).